src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SyntaxKindEx.cs

```diff
@@ -15,18 +15,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Formatting
     [SuppressMessage("CodeQuality", "IDE0051:Remove unused private members", Justification = "Private constants are used for static assertions.")]
     internal static class SyntaxKindEx
     {
-        public const SyntaxKind ImplicitObjectCreationExpression = (SyntaxKind)8659;
-        public const SyntaxKind RelationalPattern = (SyntaxKind)9029;
-        public const SyntaxKind ParenthesizedPattern = (SyntaxKind)9028;
-        public const SyntaxKind AndPattern = (SyntaxKind)9032;
-        public const SyntaxKind OrPattern = (SyntaxKind)9031;
-        public const SyntaxKind NotPattern = (SyntaxKind)9033;
-        public const SyntaxKind AndKeyword = (SyntaxKind)8439;
-        public const SyntaxKind OrKeyword = (SyntaxKind)8438;
-        public const SyntaxKind FunctionPointerType = (SyntaxKind)9056;
-        public const SyntaxKind NotKeyword = (SyntaxKind)8440;
-        public const SyntaxKind DataKeyword = (SyntaxKind)8441;
-        public const SyntaxKind RecordDeclaration = (SyntaxKind)9063;
         public const SyntaxKind ManagedKeyword = (SyntaxKind)8445;
         public const SyntaxKind UnmanagedKeyword = (SyntaxKind)8446;
```

Comment:
interesting. i thought these were older. are they not available yet?

Reply:
These are 3.8